android/java/org/chromium/chrome/browser/crypto_wallet/fragments/TxFragment.java

```diff
@@ -407,27 +410,65 @@ private void setupView(View view) {
                 mAssetPrices, mSolanaEstimatedTxFee, mFullTokenList, mNativeAssetsBalances,
                 mBlockchainTokensBalances);
 
+        if (mIsSolanaInstruction) {
+            TextView gasTxTv = view.findViewById(R.id.frag_tx_tv_gas);
+            TextView totalHeading = view.findViewById(R.id.frag_tx_tv_total_heading);
+
+            gasTxTv.setText(R.string.brave_wallet_allow_spend_transaction_fee);
+            totalHeading.setText(R.string.brave_wallet_confirm_transaction_amount_fee);
+            SolanaTxData solanaTxData = TransactionUtils.safeSolData(mTxInfo.txDataUnion);
+            if (solanaTxData != null && solanaTxData.sendOptions != null) {
+                SolanaSendTransactionOptions sendTxOptions = solanaTxData.sendOptions;
+                LinearLayout sendOptionsLl = view.findViewById(R.id.frag_tx_ll_send_options);
```

Comment:
nit: `sendOptionsLl` is hard to read, first I understood it as sendOptionsL_one and then tried to find `sendOptionsL2`. 
then I understood that `Ll` is for LinearLayout type. 
So, since there is no any other layouts here for `sendOptions`,  I would just rename it 
 `sendOptionsLl` => ` `sendOptionsLayout`
 wdyt?

Reply:
That is short for LinerarLayout i.e. `Ll` (though look like `L1`), to give an indication that it's a LinearLayout, will update.

Reply:
updated: https://github.com/brave/brave-core/pull/15593/commits/11b346d25a205d33ccba689d0f40a713ee019466